Delicate_Flowers_Vignette.qmd

```diff
@@ -75,7 +75,7 @@ Word vector analysis seemed like the obvious choice to distantly view this corpu
 
 ```{r}
 #| eval: false
-
+# Need to wrap these in an if statement so that if the file generated exists, it doesn't rerun the model, it just loads the model file (.txt or .bin respectively)
 prep_word2vec(origin="text files",destination="textfiles.txt",lowercase=T,bundle_ngrams=1)
 
 model = train_word2vec("textfiles.txt","textfiles.bin",vectors=400,threads=6,window=12,iter=3,negative_samples=5)
```

Comment:
So right here you need to wrap each of these lines, 79 and 81 in an if statement. Here's an example of what that would look like: 
```
if (!file.exists("cookbook_vectors.bin")) {
model = train_word2vec("cookbooks.txt","cookbook_vectors.bin",vectors=150,threads=8,window=12,iter=5,negative_samples=0)
} else model = read.vectors("cookbook_vectors.bin")
```

Reply:
In fact, if you wanted you don't even need the rest of this code, all you need is: 
```model = read.vectors("textfiles.bin") ```
(Thats how I got your code to render locally so I could read it as an html doc.